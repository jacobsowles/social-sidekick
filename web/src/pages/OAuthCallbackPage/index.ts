import OAuthCallbackPage from './OAuthCallbackPage';
export default OAuthCallbackPage;
