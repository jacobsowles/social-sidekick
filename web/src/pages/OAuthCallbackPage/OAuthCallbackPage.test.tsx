import React from 'react';

import { init, shallow, toJson } from '@tests/component-test-base';
import OAuthCallbackPage from '@pages/OAuthCallbackPage';

describe('OAuthCallbackPage', () => {
  init(() => <OAuthCallbackPage />, 'div');

  it('should render the component without crashing', () => {
    toJson(shallow()).should.matchSnapshot();
  });
});
