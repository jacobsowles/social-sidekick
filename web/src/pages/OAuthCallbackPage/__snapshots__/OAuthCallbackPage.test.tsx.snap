// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OAuthCallbackPage should render the component without crashing 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
